utils/create_dummy_models.py

```diff
@@ -410,7 +410,10 @@ def convert_processors(processors, tiny_config, output_folder, result):
         elif isinstance(processor, ProcessorMixin):
```

Comment:
A few updates are necessary to make the script work (in general, as well as for blip and blip_2)